akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -241,5 +247,8 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
   val serializerByIdentity: Map[Int, Serializer] =
     Map(NullSerializer.identifier -> NullSerializer) ++ serializers map { case (_, v) ⇒ (v.identifier, v) }
 
+  private def shouldWarnAboutJavaSerializer(enabled: Boolean, serializedClass: Class[_], serializer: Serializer) =
```

Comment:
perhaps use the `warnAboutJavaSerialization` flag directly here, instead of the `enabled` param?